.gitignore

```diff
@@ -6,4 +6,5 @@
 **/**/__pycache__
 .vscode/
 .pytest_cache
-src.egg-info
\ No newline at end of file
+src.egg-info
+venv/
```

Comment:
Please add a new line here

Reply:
Apart from the linting issues ,i needed you to help me look at the logic